prometheus/map.jinja

```diff
@@ -1,16 +1,96 @@
-{% set server = salt['grains.filter_by']({
-    'default': {
-        'dir': {
-            'config': '/srv/volumes/prometheus',
-            'config_in_container': '/opt/prometheus/config'
-        },
-    },
-}, merge=salt['pillar.get']('prometheus:server')) %}
-
-{% set alertmanager = salt['grains.filter_by']({
-    'default': {
-        'dir': {
-            'config': '/srv/volumes/prometheus',
-        },
-    },
-}, merge=salt['pillar.get']('prometheus:alertmanager')) %}
+
+{%- set version = pillar.prometheus.get('version', '1.6.2') %}
+
+{%- load_yaml as base_defaults %}
+{%- if pillar.docker is defined and pillar.docker.host is defined %}
```

Comment:
Yes, but this is still wrong, because you could have docker host without prometheus in them. I recommend using engine like gitlab:runner:engine = docker | local
or something like this